### DISCUSSION TITLE
I changed the client to Chinese, and changed the avatar's name to be able to display Chinese, but the avatar's name in the scene is garbled? Ask for solutions

### DISCUSSION THREAD

**Discussion by justinzm** [mozilla/hubs#5353] [Hubs Cloud Questions] [answered]
I changed the client to Chinese, and changed the avatar's name to be able to display Chinese, but the avatar's name in the scene is garbled? Ask for solutions
![111](https://user-images.githubusercontent.com/6271369/162889149-cdd15848-8b96-4e26-9716-c3529238ecf3.png)
![222](https://user-images.githubusercontent.com/6271369/162889163-aa9c4bc7-a2bf-4b16-94ae-1ee6ba7f09f3.png)

**Comment by netpro2k** [comment, marked as answer]
You will need to use a font that is capable of displaying the characters you want. See troika-text.js
